src/DragAndDropSupportWidget/widget/DroppableWidget.js

```diff
@@ -1,117 +1,67 @@
-
-        // dijit._WidgetBase.postCreate is called after constructing the widget. Implement to do extra setup work.
         postCreate: function () {
-//            console.log(this.id + '.postCreate');
-            
-//            this.domNode.appendChild(dom.create('span', { 'class': 'DroppableWidget-message' }, this.messageString));
-
+            // console.log(this.id + '.postCreate');
             this._setupEvents();
         },
 
-        // mxui.widget._WidgetBase.update is called when context is changed or initialized. Implement to re-render and / or fetch data.
         update: function (obj, callback) {
-//            console.log(this.id + '.update');
-
+            // console.log(this.id + '.update');
             this._contextObj = obj;
-            this._resetSubscriptions();
-            this._updateRendering();
 
             callback();
         },
 
-        // mxui.widget._WidgetBase.enable is called when the widget should enable editing. Implement to enable editing if widget is input widget.
-        enable: function () {
-
-        },
-
-        // mxui.widget._WidgetBase.enable is called when the widget should disable editing. Implement to disable editing if widget is input widget.
-        disable: function () {
-
-        },
-
-        // mxui.widget._WidgetBase.resize is called when the page's layout is recalculated. Implement to do sizing calculations. Prefer using CSS instead.
-        resize: function (box) {
-
-        },
-
-        // mxui.widget._WidgetBase.uninitialize is called when the widget is destroyed. Implement to do special tear-down work.
-        uninitialize: function () {
-            // Clean up listeners, helper objects, etc. There is no need to remove listeners added with this.connect / this.subscribe / this.own.
-        },
-
         _setupEvents: function () {
-            var args = {},
-                node,
-                thisObj = this;
-            
-            if (this.acceptSelector) {
-                args.accept = this.acceptSelector;
+            var self = this;
+            // http://api.jqueryui.com/droppable/
+            var args = {
+                accept: this.acceptSelector || "*",
+                reedy: true,
+                hoverClass: this.dropTargetHoverClass,
+                drop: function (event, ui) {
+                    self._drop(event, ui);
+                }
             }
-            args.greedy = true;
-            args.hoverClass = this.dropTargetHoverClass;
-            args.drop = function (event, ui) {
-                thisObj._drop(event, ui);
-            };
+            var node;
             if (this.dropTargetSelector) {
                 node = $(this.dropTargetSelector);
             } else {
                 node = $(this.domNode.parentElement);
             }
             node.addClass(this.dropTargetClass).droppable(args);
-         
         },
-        
+
         _drop: function (event, ui) {
-            var droppedObjectType,
```

Comment:
As per convention, variables are to be defined at the top of the function rather than inline. Why did you change this?

Reply:
Good question. Good code is written for readability, low maintainable and preventing bugs. This convention is correct preventing accidentally creating variables that might out of scope later on. Though modern IDE will warn you when making such mistakes. Especially when you code "strict" change are less. Noising moving the stuff around though keeping the order makes it less mind boggling. 
https://www.w3schools.com/js/js_hoisting.asp

In this case the less code makes it easier to read.